src/libraries/System.Net.Security/src/System/Net/HelperAsyncResults.cs

```diff
@@ -20,10 +20,6 @@ namespace System.Net
     //
     internal class AsyncProtocolRequest
```

Comment:
Are you planning to do subsequent cleanup to remove this in this PR, or in a later PR, or just leave that as a follow-up to be done later?

Reply:
It seems like AsyncProtocolRequest is still used in NegotiateStream code. I don't know if it needs to but I do not have any plans for removal at the moment.